src/spacesuit.rs

```diff
@@ -1,15 +1,11 @@
 #![allow(non_snake_case)]
 
-use bulletproofs::r1cs::{Prover, R1CSProof, Verifier};
+use bulletproofs::r1cs::{Prover, R1CSError, R1CSProof, Verifier};
 use bulletproofs::{BulletproofGens, PedersenGens};
-use curve25519_dalek::scalar::Scalar;
-use error::SpacesuitError;
 use gadgets::transaction;
 use merlin::Transcript;
-use rand::CryptoRng;
-use rand::Rng;
-use subtle::{ConditionallySelectable, ConstantTimeEq};
-use value::*;
+use rand::{CryptoRng, Rng};
+use value::{CommittedValue, ProverCommittable, Value, VerifierCommittable};
 
 pub struct SpacesuitProof(R1CSProof);
```

Comment:
(Note for later)

We probably don't need SpacesuitProof and prove/verify methods exported at all: the user of this crate (zkvm) will have its own transcript and CS instance and only use `spacesuit::cloak()` call to build constraints for the cloak mixer. But these are useful for tests/benchmarks, right? Can we somehow relocate them into `tests` module or something?

Reply:
yeah I mostly made these for tests and benchmarks, I didn't know at the time how the `spacesuit` crate would be getting called externally. that sounds good for a future PR!